packages/react-ux-capture-example/src/Lazy.jsx

```diff
@@ -17,13 +16,8 @@ export default class Lazy extends React.Component {
 	}
 	render() {
 		if (!this.state.loaded) {
-			return null;
+			return <React.Fragment>{this.props.fallback}</React.Fragment> || null;
```

Comment:
Also not a blocker, but to make this more user-friendly and organized as a reference implementation, I'd consider breaking out the directories one more level:

- `src/foo/Foo.jsx` - all components used only by the Foo page
- `src/shared/Lazy.jsx` - all shared components

The `shared` directory can be for components like Lazy that are used in multiple pages. The `src/` directory would be the actual view/page components, with a directory per page.

Or, some similar structure. The idea is also if we add test files, it becomes less friendly to have only a single src directory with many files.

Reply:
Yeah, components are piling up, need to group them somehow.